model/functions.php

```diff
@@ -660,4 +660,280 @@ function getAllColumbariumRelatedDataWithFilter(ColumbariumFilter $filter)
+            $statement->bindValue(':title', null, PDO::PARAM_NULL);
+        }
+        else
+        {
+            $statement->bindValue(':title', $contact->title);
+        }
+        $statement->bindValue(':phoneNumber', $contact->phoneNumber);
+        $success = $statement->execute();
+        $statement->closeCursor();
+        if($success)
+        {
+            $response->addResult(True);
+        }
+        else{
+            $response->addError("Failed to insert Contact.");
+        }
+    } catch (PDOException $e) {
+        $errorMessage = $e->getMessage();
+        $response->addError($errorMessage);
+    }
+    return $response;
+}
+
+function insertNicheType(ToTypeTable $nicheType)
+{
+    $response = new Response();
+    try{
+        $db = connection::getInstance();
+        $con = $db->get_connection();
+        $query = "INSERT INTO niche_types (TYPE)
+                    VALUE (:nicheType);";
+        $statement = $con->prepare($query);
+        $statement->bindValue(':nicheType', $nicheType->type);
+        $success = $statement->execute();
+        $statement->closeCursor();
+        if($success)
+        {
+            $response->addResult(True);
+        }
+        else
+        {
+            $response->addError("Failed to insert Niche Type");
+        }
+    } catch (PDOException $e) {
+        $errorMessage = $e->getMessage();
+        $response->addError($errorMessage);
+    }
+    return $response;
+}
+
+function insertTombSectionLetter(ToSectionLetter $sectionLetter)
+{
+    $response = new Response();
+    try{
+        $db = connection::getInstance();
+        $con = $db->get_connection();
+        $query = "INSERT INTO tomb_section_letters (SECTION_LETTER)
+                    VALUE (:sectionLetter);";
+        $statement = $con->prepare($query);
+        $statement->bindValue(':sectionLetter', $sectionLetter->letter);
+        $success = $statement->execute();
+        $statement->closeCursor();
+        if($success)
+        {
+            $response->addResult(True);
+        }
+        else
+        {
+            $response->addError("Failed to insert a Tomb Section Letter.");
+        }
+    } catch (PDOException $e) {
+        $errorMessage = $e->getMessage();
+        $response->addError($errorMessage);
+    }
+    return $response;
+}
+
+function insertColumbariumSectionLetter(ToSectionLetter $sectionLetter)
+{
+    $response = new Response();
+    try{
+        $db = connection::getInstance();
+        $con = $db->get_connection();
+        $query = "INSERT INTO columbarium_section_letters (SECTION_LETTER)
+                    VALUE (:sectionLetter);";
+        $statement = $con->prepare($query);
+        $statement->bindValue(':sectionLetter', $sectionLetter->letter);
+        $success = $statement->execute();
+        $statement->closeCursor();
+        if($success)
+            $response->addResult(True);
+        else
+            $response->addError("Failed to insert Columbarium Section Letter.");
```

Comment:
Lines 930 - 933
Wrap if else in curly braces. It is unnecessary in some parts, but here it makes it more sense to make it readable for what we are doing with the success and failure of the function.